src/ScanItems.js

```diff
@@ -239,11 +239,15 @@ class ScanItems extends React.Component {
     ] = [],
   }) => {
     const itemError = (!parameters || !parameters.length)
-      ? {
-        barcode: <FormattedMessage id="ui-checkout.unknownError" />,
-        _error: 'unknownError',
-      }
-      : {
+      ? (parameters.length === 0)
+        ?
```

Comment:
It is hard to reason about more than one ternary really. I would recommend changing that implementation to use `if` statements.